modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/v2_0_0/SXPBlueprintUpgradeProcess.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.search.experiences.internal.upgrade.v2_0_0;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * @author Gustavo Lima
+ */
+public class SXPBlueprintUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		String portletIdQuoted = StringUtil.quote(
+			"%com_liferay_search_experiences_web_internal_blueprint_options_" +
+				"portlet_SXPBlueprintOptionsPortlet_INSTANCE_%");
+
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				StringBundler.concat(
+					"select portletPreferencesId from PortletPreferences ",
+					"where portletId like ", portletIdQuoted))) {
+
+			ResultSet resultSet = preparedStatement1.executeQuery();
+
+			_upgradeSXPBlueprintOptionsPortlets(resultSet);
+		}
+	}
+
+	private void _upgradeSmallValueAndName(
+			String portletPreferencesIdQuoted, ResultSet resultSet2)
```

Comment:
Just resultSet instead of resultSet2. Please check all the numbered variables in this class as they seem to start from random numbers. 

I assume this happened because of copy & paste, when refactoring the code and splitting the logic.